src/caf/toolkit/translation.py

```diff
@@ -580,19 +588,24 @@ def pandas_matrix_zone_translation(
         "translation_to_col": translation_to_col,
         "translation_factors_col": translation_factors_col,
         "translation_dtype": translation_dtype,
-        "check_totals": check_totals,
+        "check_totals": False,
```

Comment:
It looks like the 'check_totals' will still raise an error if matrices don't match, rather than a warning

Reply:
Added issue #112